iOS Example Frame/iOS Example Frame/Factory/Factory.swift

```diff
@@ -11,8 +11,11 @@ import Checkout
 import UIKit
 
 enum Factory {
+
+  // swiftlint:disable force_unwrapping
   static let successURL = URL(string: "https://httpstat.us/200")!
   static let failureURL = URL(string: "https://httpstat.us/403")!
```

Comment:
should we just add // swiftlint:disable next force_unwrapping for line 17 and 15 as well instead of adding enable again on 18.

Reply:
Sure, updated. Suppose it does make it less likely stuff could get added inside the block by mistake 👍